WcaOnRails/app/views/search_results/index.html.erb

```diff
@@ -86,5 +101,8 @@
     <% if @posts.empty? %>
       <h2>No posts found for <%= wca_highlight(@omni_query, @omni_query) %></h2>
     <% end %>
+    <% if @regulations.empty? %>
+      <h2>No Regulation or Guideline found for <%= wca_highlight(@omni_query, @omni_query) %></h2>
```

Comment:
nit: for consistency, consider pluralizing? That is, "No Regulations or Guidelines found for ..."